hashicorp-vault-dynamic-secret-ysql-plugin/tear-down.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Stop all the docker containers
+docker stop $(docker ps -a -q)
```

Comment:
Does the test launch and run inside Docker containers?

Reply:
The cluster is created on docker.